.env.sample

```diff
@@ -7,10 +7,11 @@
 HARBOR_USER='admin'
 HARBOR_PASSWORD=''
 HARBOR_BASE_URL='http://127.0.0.1:8083/api/v2.0'
+HARBOR_BASE_REPO_URL='harbor.dev.eks.otomi.cloud'
```

Comment:
Should we have this FQDN in here? Or should we point to localhost with K8s PF?

Reply:
docker needs a FQDN. Is this variable used to access docker images @mojtabaimani ? What is this used for? If this is unrelated to docker we should indeed use the proxy url. This file is for devving...

Reply:
yes docker needs FQDN

Reply:
It is used to create correct pull secret